File: nodejs/ip_address.js
const cassandra = require('cassandra-driver');


function init_connection(){
    var connection = {}
    connection.ip_address = '127.0.0.1';
    connection.dc = 'Cassandra'
    connection.path_to_creds=''
    connection.client = new cassandra.Client({ 
        contactPoints: [connection.ip_address],
        localDataCenter: connection.dc 
    });
    return connection
}

connection = init_connection()

module.exports = connection;

File: python/ip_address.py
#!/usr/bin/env python3      
from dse.cluster import Cluster
from dse.query import named_tuple_factory

class Connection:
    def __init__(self):
        self.ip_address='127.0.0.1'
        self.path_to_creds=''
        self.cluster = Cluster([self.ip_address])
        self.session = self.cluster.connect()
        self.session.row_factory = named_tuple_factory